presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PushPartialAggregationThroughExchange.java

```diff
@@ -215,6 +215,7 @@ private PlanNode split(AggregationNode node, Context context)
                 node.getSource(),
                 intermediateAggregation,
                 node.getGroupingSets(),
+                ImmutableList.of(),
```

Comment:
you shouldn't create non-streaming partial aggregation for streaming aggregation. This will hurt performance